test/property.test.js

```diff
@@ -125,6 +207,20 @@ describe('loopback:property generator', function() {
     });
   });
 
+  it('should fail to create with unsupported type', function (done) {
+    var propertyGenerator = givenPropertyGenerator();
+    helpers.mockPrompt(propertyGenerator, {
+      model: 'Car',
+      name: 'something',
+      type: 'unknown',
```

Comment:
I don't think this is possible to achieve via CLI UI, because the `type` is picked from a list. One can specify `customType` with arbitrary value, that's a valid use case that should be supported as I commented earlier.